### ISSUE TITLE
HTML escaping should not happen on dot strings

### ISSUE THREAD

**Issue by Clashsoft** [nidi3/graphviz-java#147]
Our library builds dotfiles as strings and then passes them to Graphviz. A typical dotfile looks like this:

```dot
digraph H {
studyRight1 [
   shape=plaintext
   fontsize="10"
   label=<
     <table border='0' cellborder='1' cellspacing='0'>
       <tr><td><u>studyRight :StudyRight</u></td></tr>
       <tr><td>  description = "&lt;i&gt;Greatest Ever&lt;/i&gt;"<br  align='left'/>  id = "studyRight"<br  align='left'/></td></tr>
     </table>
  >];
}
```

When pasting this on viz-js.com for example, it looks like this (desired result):

![Screenshot 2020-03-05 at 18 43 32](https://user-images.githubusercontent.com/4145923/76009262-38c8fc00-5f11-11ea-8d36-29fc2587a9ce.png)

However, with Graphviz as of v0.11.1 and up it renders like this:

![specialChars](https://user-images.githubusercontent.com/4145923/76008571-25696100-5f10-11ea-8c0c-9eb260df2d0a.png)

This is the code we use to render it:

```java
// format = Format.PNG or .SVG
Graphviz.fromString(dotString).render(format).toFile(new File(diagramFileName));
```

I already adapted the code that generates the dotfile to remove the manual HTML escaping. This produced the following result:

![specialChars](https://user-images.githubusercontent.com/4145923/76008640-416d0280-5f10-11ea-8556-5f65428ff011.png)

---

The change was introduced in f5125a6. I believe this should be reverted partially such that HTML escaping is done for graphs build with this library's model (e.g. `node("A&B")`), but not for entire dotfile strings.
For those, HTML escaping should be the user's job. Otherwise there is no way to circumvent it.

---

Context: https://github.com/fujaba/fulibTools/issues/3

**Comment by nidi3** [comment]
I think you are right, that is a little bit of overescaping.

**Comment by nidi3** [comment]
fixed in version `0.15.1`
